src/screens/Profile/Profile.js

```diff
@@ -215,6 +215,25 @@ class Profile extends React.Component<Props, State> {
     });
   };
 
+  handleChangeUseBiometrics = (value) => {
+    const { changeUseBiometrics } = this.props;
+    changeUseBiometrics(value);
+    if (value) {
+      return TouchID.isSupported()
+        .then(biometryType => {
+          if (biometryType === 'FaceID') {
+            FaceId.request();
+            this.setBiometricState(value);
+          } else if (biometryType === 'TouchID') {
+            this.setBiometricState(value);
+          } else if (biometryType === true) {
+            this.setBiometricState(value);
+          }
```

Comment:
If `biometryType` is none of the listed types, `this.setBiometricState(value);` will never be called, is that right? I'm curious because if it fails, it will be called.